providers/manage.rb

```diff
@@ -178,3 +180,18 @@ def manage_home_files?(home_dir, user)
     true
   end
 end
+
+# This is a workaround to include recipe 'chef-vault' in LWRP.
+def chef_vault_item(bag, item)
```

Comment:
i don't think this is needed with 1.2.0+ of the chef-vault cookbook

https://github.com/opscode-cookbooks/chef-vault/blob/v1.2.0/libraries/chef_vault_item.rb#L23-L45